lib/vty.c

```diff
@@ -312,6 +312,13 @@ int vty_json_no_pretty(struct vty *vty, struct json_object *json)
 	return vty_json_helper(vty, json, JSON_C_TO_STRING_NOSLASHESCAPE);
 }
 
+void print_empty_json_obj(struct vty *vty)
+{
+	json_object *json = json_object_new_object();
```

Comment:
IMO, we should pass the object as an argument to properly free the memory that is in some cases allocated but not freed before return or so. If NULL is passed, we can create the object and print it, otherwise, use what's received to free properly.

Reply:
If developer has created a new JSON object, then they can use vty_json() to print and free the JSON object before returning.

There are cases where the show command must return empty JSON object even before the JSON object is created. In such cases, developer can call this helper function to create a new JSON object, print and free it.